code/modules/projectiles/gun_attachables.dm

```diff
@@ -689,7 +690,10 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	var/zoom_offset = 11
 	///how many tiles to increase the users view box
 	var/zoom_viewsize = 12
-	scoped_accuracy_mod = SCOPE_RAIL
+	///can this scope move while zooming ? default of no, scopes with higher viewrange than 7 cause maptick lag on moving according to tivi.
+	var/zoom_movement = FALSE
```

Comment:
also add a CRASH()  for scopes if they set it too high

Reply:
if zoom movement with viewrange high ?

Reply:
if they have both this and a viewrange higher than default add a CRASH to the zoom proc